File: scala-akka-http-server/src/main/scala/org/openapitools/server/model/CodedErrorBaseAllOf.scala
package org.openapitools.server.model


/**
 * @param result  for example: ''null''
 * @param msg  for example: ''null''
 * @param code A string that identifies the error.  for example: ''null''
*/
final case class CodedErrorBaseAllOf (
  result: Option[AnyType],
  msg: Option[AnyType],
  code: Option[String]
)

File: fsharp-giraffe-server/OpenAPI/src/model/CodedErrorBaseAllOf.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.AnyType

module CodedErrorBaseAllOf = 

  //#region CodedErrorBaseAllOf


  type CodedErrorBase_allOf = {
    Result : AnyType;
    Msg : AnyType;
    Code : string;
  }
  //#endregion
  